### ISSUE TITLE
Slate template broken

### ISSUE THREAD

**Issue by terry90** [zipmark/rspec_api_documentation#338]
Hey,

I just ran into an issue with the current Slate version.
The template for Slate generates some blocks like this:

    ### Request

    #### Endpoint
    ```
    GET /v1/app_configs
    Authorization: Token token=qyKRKqDwtFs7QgUNjNgo
    Accept: application/json
    ```

or

    ### Response
    
    ```
    Content-Type: application/json; charset=utf-8
    200 OK
    ```

The problem is, no lang is specified for these blocks, Slate tries to interpret them but throws this:

```
NoMethodError at /
undefined method `split' for nil:NilClass

Ruby	/home/xxx/xx/lib/multilang.rb: in block_code, line 3
Web	GET localhost/
```

When I edit my Markdown file and replace \``` by ```plaintext it stops crashing.

**Comment by sweinertjr** [comment]
I'm experiencing the same issue. I'm not sure how to stop it from crashing though

**Comment by oestrich** [comment]
Happy to accept a PR if you can fix the template or whatever else is wrong with it. I don't use Slate so I can't help much.

**Comment by terry90** [comment]
@sweinertjr You can simply edit the template to add `plaintext` next to the declaration of code block.
E.g: 


    ### Response
    
    ```
    Content-Type: application/json; charset=utf-8
    200 OK
    ```

Becomes:

    ### Response
    
    ```plaintext
    Content-Type: application/json; charset=utf-8
    200 OK
    ```

**Comment by terry90** [comment]
Fixed with PR #346
